ingestion-beam/src/test/java/com/mozilla/telemetry/posthog/PosthogEventTest.java

```diff
@@ -19,15 +19,20 @@ public void testToJson() throws JsonProcessingException {
     experiments.put("experiment2", "branch_b");
     PosthogEvent eventWithExtras = PosthogEvent.builder().setUserId("test").setSampleId(1)
         .setEventType("bookmark.event").setPlatform("firefox-desktop").setTime(1738620582500L)
-        .setEventExtras("{\"metadata1\":\"extra\",\"metadata2\":\"more_extra\"}")
+        .setEventExtras("{" + "\"metadata1\":\"extra\"," + "\"metadata2\":\"more_extra\","
```

Comment:
Aren't extra values always strings? Or can they be anything and just get cast to string when writing to bigquery?

Reply:
You're right, based on https://mozilla.github.io/glean/book/reference/metrics/event.html#events _Values are one of string, boolean, or quantity type and are converted to string for transmission_